### PR TITLE
Update description and add flag to SessionScheduleDto to denote include / exclude location groups.

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
## What does this pull request do?

Updates description and adds flag to SessionScheduleDto

## What is the intent behind these changes?

VB-3452 improvements

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/CreateSessionTemplateDto.kt [hunk 1]
```diff
@@ -59,6 +59,6 @@ data class CreateSessionTemplateDto(
   @Schema(description = "list of group references for allowed prisoner incentive levels", required = false)
   val incentiveLevelGroupReferences: List<String>? = listOf(),
 
-  @Schema(description = "Include location group types if true, exclude if false.", required = true)
+  @Schema(description = "Determines behaviour of location groups. True will mean these location groups are included, false means they will be excluded.", required = true)
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/CreateSessionTemplateDto.kt:62]: updated description as discussed

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt [hunk 2]
```diff
@@ -448,6 +448,7 @@ open class SessionService(
       sessionTemplateReference = sessionTemplate.reference,
       sessionTimeSlot = SessionTimeSlotDto(startTime = sessionTemplate.startTime, endTime = sessionTemplate.endTime),
       capacity = SessionCapacityDto(sessionTemplate),
+      areLocationGroupsIncluded = sessionTemplate.includeLocationGroupType,
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt:451]: setting areLocationGroupsIncluded on SessionScheduleDto

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/SessionScheduleDto.kt [hunk 3]
```diff
@@ -24,6 +24,9 @@ data class SessionScheduleDto(
   @Schema(description = "visit type", example = "Social", required = true)
   val visitType: VisitType,
 
+  @Schema(description = "Determines behaviour of location groups. True will mean the location groups are included, false means they are excluded.", required = true)
```
Comment (dhirajshettymoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/SessionScheduleDto.kt:27]: adding new flag to SessionScheduleDto as discussed

**Review by tpmcgowan — APPROVED** [review]
LGTM

**Pull request merged**